bin/key_server.js

```diff
@@ -5,11 +5,23 @@
 var config = require('../config').getProperties()
 var jwtool = require('fxa-jwtool')
 
+var log = require('../lib/log')(config.log.level)
+var getGeoData = require('../lib/geodb')(log)
+
 function main() {
-  var log = require('../lib/log')(config.log.level)
   var metricsContext = require('../lib/metrics/context')(log, config)
   log.setMetricsContext(metricsContext)
 
+  // Force the geo to load and run at startup, not waiting for it to run on
+  // some route later.
+  if (config.env === 'prod' || config.env === 'stage') {
```

Comment:
@jrgm should we just do this for all envs? 

* to avoid more branches between dev and prod
* it will fail safely if no db, etc